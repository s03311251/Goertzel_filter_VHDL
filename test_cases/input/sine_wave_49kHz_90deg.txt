3FFF
3E7E
3A1F
334A
2AA6
2101
1744
0E59
0717
022C
0010
00F5
04C5
0B24
137A
1CFD
26C8
2FF0
3799
3D09
3FBE
3F77
3C3B
3657
2E58
2501
1B31
11D5
09CD
03DD
0091
003A
02E0
0844
0FE2
1905
22CF
2C56
34B4
3B1F
3EFE
3FF2
3DE5
3908
31D1
28ED
1F32
158A
0CDE
05FE
0191
0000
0172
05C3
0C8C
1529
1ECB
288A
317B
38C8
3DC0
3FEB
3F17
3B55
3502
2CB5
2336
1969
103B
0889
030C
0047
007E
03AC
0984
1179
1ACB
249B
2DFC
360C
3C0A
3F64
3FCB
3D34
37DE
3049
272D
1D63
13D9
0B72
04FB
010E
000A
0208
06D7
0E03
16E1
209A
2A45
32F8
