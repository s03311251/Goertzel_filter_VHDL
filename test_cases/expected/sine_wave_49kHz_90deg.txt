FFF9CA20
FFF44B00
